src/Bicep.Core.Samples/Files/InvalidExpressions_LF/main.diagnostics.bicep

```diff
@@ -335,7 +335,7 @@ var invalidPropertyCallOnInstanceFunctionAccess = a.b.c.bar().baz
 var invalidInstanceFunctionAccess = a.b.c.bar()
 //@[36:37) [BCP057 (Error)] The name "a" does not exist in the current context. |a|
 var invalidInstanceFunctionCall = az.az()
-//@[37:39) [BCP107 (Error)] The function "az" does not exist in namespace "az". |az|
+//@[37:39) [BCP109 (Error)] The type "az" does not contain function "az". |az|
```

Comment:
this change in diagnostic is quite odd...

Reply:
Namespaces are special object types, so this is a result of the change to avoid binding `az.<func>()` at bind time, and instead using dynamic method resolution (which is required for method lookups on all other object types, as they can't be resolved at bind time).

I'll try and special-case this to restore the old error message, as it's more informative.